src/vs/code/electron-main/main.ts

```diff
@@ -481,7 +481,8 @@ class CodeMain {
 				args.waitMarkerFilePath = waitMarkerFilePath;
 			}
 		}
-
+		addArg(process.argv, '--autoplay-policy', 'no-user-gesture-required');
```

Comment:
since the value is static, where is a better place for this?

Reply:
Given this is a chromium flag, you can add it in https://github.com/microsoft/vscode/blob/93026118c5c68f54220d5a39993a5e29a8df0029/src/main.js#L174-L262 instead.

Reply:
lmk if this is not where you expected it to be https://github.com/microsoft/vscode/blob/fbeb172116a40aff96805bfe636e622e86ce541b/src/main.js#L201